lib/managers/remote_config_manager.dart

```diff
@@ -1,11 +1,18 @@
 import 'package:api/dependency_injection.dart';
 import 'package:api/models/app/managers/remote_config.dart';
+import 'package:globo_fitness/managers/connection_check_status_manager.dart';
 
 final RemoteConfigManager _remoteConfigManager =
     DependecyInjection.instance.get<RemoteConfigManager>();
 
 Future<void> activateRemoteConfig() async {
-  await _remoteConfigManager.initializeRemoteConfig();
-  _remoteConfigManager.setDefaultParams();
-  _remoteConfigManager.launchLoadingStrategy();
-}
+  ConnectionCheckStatusManager connectionCheckStatusManager =
+      ConnectionCheckStatusManager();
+  if (await connectionCheckStatusManager.hasInternetConnection()) {
+    await _remoteConfigManager.initializeRemoteConfig();
+    _remoteConfigManager.setDefaultParams();
+    _remoteConfigManager.launchLoadingStrategy();
+  } else {
+    connectionCheckStatusManager.activateConnectionManager();
```

Comment:
connectionCheckStatusManager.activateConnectionManager() isn't needed because it's already call in main_common

Reply:
Also, same thing use the dependency injection to get the connectionCheckStatusManager